README.md

```diff
@@ -0,0 +1,44 @@
+# Welcome to PubDoc
+
+## TL;DR
+The goal of this repository is twofold:
+1. To make documentation better for the reader.
+2. To make _creating_ documentation a better experience for the writer.
+
+## How to Use
+This repository is currently very "bare bones." To run the tool and interface with Confluence,  
+you may need to contact an IT administrator to obtain permissions to access Confluence data.  
+
+Visit [this link](https://id.atlassian.com/manage-profile/security/api-tokens) to create your API token.  
+Next, create a `.env` file with the following contents:
+
+### `.env` File Example
+```env
+CONFLUENCE_USERNAME=<username>
+CONFLUENCE_API_KEY=<api key>
+CONFLUENCE_DOMAIN=<your company>.atlassian.net
+CONFLUENCE_SPACEID=<A space you don't care about littering in>
+```
+To get a spaceID of a space. Go to the space then click on "space settings" in the top left. Then run this in the browser to get the spaceID. `https://YOURURL.atlassian.net/wiki/rest/api/space/KEY`
+
+## Pontifications & Justifications
+If you're interested in why we're working on this, here's some context:  
+
+Want to feel depressed? Go look at a Confluence documentation page. You can see who has visited or read the page.  
+You'll likely notice that the number of viewers per page is remarkably low. Who's to blame—the reader or the writer?  
+Who's to say. Regardless, one thing is certain: the documentation could be better.  
```

Comment:
"Who's to say." -> "Who's to say?"

Reply:
I'm technically not trying to ask a question here. 

Rather I'm stating a common rhetorical question. But it is a statement. Intended to be read as _mumbling_ or _an aside comment_.

![8db6bc0205645041dac858f5edd31c98](https://github.com/user-attachments/assets/4164fc07-b868-44eb-ae42-62f465919066)

I'll change it though idc. _erm actually applying correct grammer here removes the intended effect of the written language._